libs/gi/page-team/src/CharacterDisplay/Build/BuildReal.tsx

```diff
@@ -212,7 +230,7 @@ function BuildEditor({
     const { name, description } = newBuild
     setName(name)
     setDesc(description)
-  }, [database, buildId])
+  }, [database, buildId, teamId, teamCharId])
```

Comment:
I am not sure if adding these to the dependency is necessary, can you explain?

Reply:
Oh woops, those aren't supposed to be there anymore. Must've snuck back while I was solving the merge conflicts during rebase